src/Client/Logger.java

```diff
@@ -83,8 +87,22 @@ public static void start() {
   /** Deletes all log files that exceed the MAX_LOG_FILES count */
   public static void purgeOldestLogFiles() {
     File logsDir = new File(Settings.Dir.LOGS);
-    File[] logFiles = logsDir.listFiles();
-    if (logFiles != null && logFiles.length > MAX_LOG_FILES) {
+
+    File[] fileList = logsDir.listFiles();
+    if (fileList == null) {
+      return;
+    }
+
+    File[] logFiles =
+        Arrays.stream(fileList)
+            .filter(
+                f ->
+                    f.isFile()
+                        && f.getName().startsWith(LOG_FILE_PREFIX)
+                        && f.getName().endsWith(LOG_FILE_EXTENSION))
```

Comment:
```java
&& f.getName().length() == LOG_FILE_PREFIX.length() + LOG_FILE_EXTENSION.length() + "yyyy-MM-dd_HH-mm-ss".length()
```
to reduce chance of deleting a file named `rscplus_2023-10-10_12-00-00-veryInterestingLogDontDelete.log`.

Reply:
,,,,,,,,,,,,,,,,.

Reply:
Yeah okay lol, done